client/tests/e2e/utils/show-cursor.js

```diff
@@ -0,0 +1,152 @@
+(() => {
```

Comment:
@Rad14nt What does this do exactly and why do we need this? Is this something that a library could do for us?